zcash_client_backend/CHANGELOG.md

```diff
@@ -32,9 +58,6 @@ and this library adheres to Rust's notion of
     `map_internal_account_note` and `map_ephemeral_transparent_outpoint` and
     `internal_account_note_transpose_option` methods have consequently been
     removed.
-- `zcash_client_backend::data_api::WalletRead::get_known_ephemeral_addresses`
-  now takes a `Range<zcash_transparent::keys::NonHardenedChildIndex>` as its
-  argument instead of a `Range<u32>`
```

Comment:
When did this get into the wrong section? It's not added by this PR which means it must have been added in a PR that was included in the latest release. Are we certain this is not part of 0.17.0?

Reply:
Will check.

Reply:
The actual change was introduced in 6556505e0601a2fedaf0b43dded82d3b6c2f29a4, but the errant log entry was added in 5290d13942d4a3f149b86459ee3556d1dac196d8 - that change was factored out from this PR and I missed that the changelog contained this error.